istioctl/pkg/auth/upgrader.go

```diff
@@ -214,24 +214,51 @@ func (ug *Upgrader) convertClusterRbacConfig(authzPolicies *model.AuthorizationP
 	if clusterRbacConfig == nil {
 		return fmt.Errorf("no ClusterRbacConfig found")
 	}
+	// RbacConfig_ON_WITH_INCLUSION doesn't require the config root namespace.
+	if clusterRbacConfig.Mode == rbac_v1alpha1.RbacConfig_ON_WITH_INCLUSION {
```

Comment:
Why are we doing this separately from the following `switch clusterRbacConfig.Mode`? Can you merge them together.

Reply:
RbacConfig_ON_WITH_INCLUSION doesn't need the root namespace, so I thought doing this would reduce duplicating code.